app/models/commit_status.rb

```diff
@@ -11,7 +11,11 @@ def status
   end
 
   def status_list
-    combined_status.fetch(:statuses).map(&:to_h)
+    list = combined_status.fetch(:statuses).map(&:to_h)
+    if list.empty?
+      list << {state: 'pending', description: 'Waiting for reply from CI, ignore if none is set up'}
```

Comment:
I like the original more ... it's shorter and it tells a user that this might be a false positive